### PR TITLE
Bring in vercel-action updates

### PR DESCRIPTION
**Pull request by glitteringkatie**

### CODE REVIEW

**Comment by glitteringkatie** [comment]
Heyyyy @gtback looks like I was never added to this repo. So two questions:

1. Can you add me and Scotty please?
2. Do you have a suggestion for what to do about the CLA above?

**Comment by gtback** [comment]
cla/check

File: package.json [hunk 1]
```diff
@@ -23,29 +23,29 @@
     "all": "npm run format && npm run lint && npm run package && npm test"
   },
   "dependencies": {
-    "@actions/core": "^1.2.6",
+    "@actions/core": "^1.10.0",
```
Comment (glitteringkatie) [inline package.json:26]: Verified this is what github recommends to get the latest get-output: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#patching-your-actions-and-workflows

**Comment by glitteringkatie** [comment]
Hey @scottybollinger I would love your review on this! Heads up that changes to `.github/workflows` and `examples` don't really pertain to us since we don't run either. However good to look over everything to make sure we both agree with brining in outside code 👍

Resolving conflicts now.

CLA policy update: https://github.com/elastic/open-source/issues/303#issuecomment-1405499576

**Comment by glitteringkatie** [comment]
Because I can't change upstream, I have a new PR that will merge all of these changes and updates dist (our one merge conflict). I am leaving this PR open for ease of review but we'll be merging https://github.com/elastic/builder/pull/5 instead.

**Comment by scottybollinger** [comment]
Also, with all of these changes and failed checks, is there any way we can test this before merging?

**Comment by glitteringkatie** [comment]
I too am confused. Hold please.

**Comment by glitteringkatie** [comment]
In the mean time, would love your eyes on this PR.

**Pull request closed without merging**